packages/orchestrator/lib/clients/client.ts

```diff
@@ -162,7 +162,7 @@ export class OrchestratorClient {
         }
     }
 
-    private async execute(props: ExecuteProps): Promise<ExecuteReturn> {
+    private async immediateAndWait(props: ExecuteProps): Promise<ExecuteReturn> {
```

Comment:
unrelated but `execute` was a bit vague. We now have an `recurring`, `immediate` and `immediateAndWait` functions